paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -282,7 +299,10 @@
                                 "signalflow_metrics_query": {
                                     "type": "string"
                                 }
-                            }
+                            },
+                            "required": [
+                                "signalflow_metrics_query"
```

Comment:
Have you and @EvanKrall already come to an agreement that this is the expected user interface for defining "custom" autoscaling stuff?

Reply:
Yes. We reached an agreement